views/assets/javascripts/tyk.js

```diff
@@ -34,3 +34,22 @@ function copyToClipboard(elemId) {
   const codeRefEl = document.getElementById(elemId);
   navigator.clipboard.writeText(codeRefEl.value);
 }
+
+function hideHelpSection() {
+	let overviewToggleButton = document.getElementsByClassName('toggle-help-section');
```

Comment:
Just an alternative to avoid using index, 

```js
let overviewToggleButton = document.getElementsByClassName('toggle-help-section')[0];
let overviewToggleSection = document.getElementsByClassName('help-section')[0];
```

Reply:
I would write it like this:
```js
let overviewToggleButton = document.querySelector('.toggle-help-section');
let overviewToggleSection = document.querySelector('.help-section');
```

Reply:
Yeah, that was my thought as well but then I learned that `getElementsByClassName` is more efficient than 
`querySelector` 

```
Function               | Live? | Type           | Time Complexity
querySelector          |       | Element        |  O(n)
querySelectorAll       |   N   | NodeList       |  O(n)
getElementById         |       | Element        |  O(1)
getElementsByClassName |   Y   | HTMLCollection |  O(1)
getElementsByTagName   |   Y   | HTMLCollection |  O(1)
getElementsByName      |   Y   | NodeList       |  O(1)
```

Reply:
@jay-deshmukh Using index in case there are multiple sections.

@ifrim why do you prefer that method ?

Reply:
`querySelector` stops after finding an element that matches the selector. The time difference is negligible.
I prefer it because it is less to write, easier to understand, and it does exactly what is needed in this case.

This type of micro optimizations should be left to the browser. That selector can easily be transformed into some other internal call by the browser. As a rule of thumb we should write code for humans :)